app/editor/src/features/content/form/ContentForm.tsx

```diff
@@ -156,10 +178,40 @@ export const ContentForm: React.FC<IContentFormProps> = ({
     [getContent],
   );
 
+  const cleanUpSignalRConnection = (connection: HubConnection) => {
+    connection?.off('Update');
+    connection?.stop();
+  };
+
   React.useEffect(() => {
+    const setUpSignalRConnection = async (id: number) => {
+      const connection = new HubConnectionBuilder()
```

Comment:
No hardcoding components to an API implementation allowed.
All API calls must be a hook created in the `/hooks/api-editor` folder.
If they interact with redux in any way, they must also have a wrapper hook in the /store/hooks` folder which will then perform the actual API call.

Reply:
Code updated, and there is no API implementation here instead of just setting up a hub connection in this case, thanks